.README/rules/sort-tags.md

```diff
@@ -8,24 +8,32 @@ Sorts tags by a specified sequence according to tag name.
 
 ##### `tagSequence`
 
-An array of tag names indicating the preferred sequence for sorting tags.
+An array of tag group objects indicating the preferred sequence for sorting tags.
+
+Each item in the array can be an object with a `tags` property set to an array
```

Comment:
"can be an object" -> "Should be an object" (or "is an")?

Reply:
Sure, changed to "should be"